R/elbo_ss.R

```diff
@@ -19,13 +19,14 @@ Eloglik_ss = function(XtX,Xty,s, var_y, n){
   return(result)
 }
 
-# expected squared residuals
+#' @title expected squared residuals
+#' @importFrom Matrix diag
 get_ER2_ss = function(XtX,Xty,s,var_y,n){
   B = s$alpha*s$mu
   XB2 = sum(t(B) * XtX%*%t(B))
 
   betabar = colSums(B)
-  d = diag(XtX)
+  d = Matrix::diag(XtX)
```

Comment:
@zouyuxin do you still need `Matrix::diag` when you have it `importFrom Matrix diag` and properly ran `roxygen2` to add it to NAMESPACE?